src/components/GameCards/GameCard.js

```diff
@@ -28,9 +32,16 @@ export default {
     },
 
     gameCardClass() {
+      let badge = '';
+
+      if (this.gameProgress === '100') {
+        badge = 'badge';
+      }
```

Comment:
consider using ternary operator instead

```suggestion
const badge = this.gameProgress === '100'
    ? 'badge'
    : '';
```